cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskPlanner.java

```diff
@@ -68,6 +68,7 @@
   private static final Logger LOG = LoggerFactory.getLogger(ExecutionTaskPlanner.class);
   private Map<Integer, SortedSet<ExecutionTask>> _interPartMoveTasksByBrokerId;
   private SortedSet<Integer> _interPartMoveBrokerIds;
+  private List<Integer> _interPartMoveBrokerIdsList;
```

Comment:
This can be turned into a local variable inside the getInterBrokerReplicaMovementTasks method.